core/src/main/java/io/grpc/ClientInterceptors.java

```diff
@@ -17,6 +17,7 @@
 package io.grpc;
 
 import com.google.common.base.Preconditions;
+import io.grpc.internal.ManagedChannelChain;
```

Comment:
FYI: io.grpc can't depend on io.grpc.internal. It seems ManagedChannelChain should just be package-private in io.grpc.